code/game/objects/items/storage/uplink_kits.dm

```diff
@@ -235,6 +237,13 @@
 			
 		if(KIT_TRAITOR_2006) //A kit so old, it's probably older than you. //This bundle is filled with the entire unlink contents traitors had access to in 2006, from OpenSS13. Notably the esword was not a choice but existed in code.
 			/obj/item/storage/toolbox/emergency/old/ancientbundle //Items fit neatly into a classic toolbox just to remind you what the theme is.
+			
+		if(KIT_RED_IS_SUS)
+			new /obj/item/clothing/head/helmet/space/syndicate(src)
+			new /obj/item/clothing/suit/space/syndicate(src)
```

Comment:
if you're expecting them to primarily use a melee weapon, it might be better to give them a nukie hardsuit, which won't slow them down while they wear it if they set it to non-EVA mode

Reply:
No, the soft suit is part of the meme you dip